ocs_ci/ocs/disruptive_operations.py

```diff
@@ -62,3 +65,28 @@ def worker_node_shutdown(abrupt):
         log.info("Abrupt Shutdown")
         if node_to_shutdown:
             nodes.start_nodes(nodes=node_to_shutdown)
+
+
+def osd_node_reboot():
+    """
+    Rebooting worker node that running OSD
+
+    Raises:
+        AssertionError: in case the ceph-tools pod was not recovered
+
+    """
+    nodes = PlatformNodesFactory().get_nodes_platform()
+    osd_nodes_names = get_osd_running_nodes()
+    osd_node_to_reboot = list()
+    for node in get_nodes():
+        node_name = get_node_name(node)
+        if node_name == osd_nodes_names[0]:
```

Comment:
doesn't [0] index alway compare the first osd node?

Reply:
yes, test should just reboot 1 OSD node